tests/testing_utils.py

```diff
@@ -251,7 +251,7 @@ def update_OptOptions_output(self, optOptions):
             optOptions[optionName] = f"{self.id()}{suffix}"
         return optOptions
 
-    def optimize(self, sens=None, setDV=None, optOptions=None, storeHistory=False, hotStart=None):
+    def optimize(self, sens=None, setDV=None, optOptions=None, storeHistory=False, hotStart=False):
```

Comment:
Was this a bug in the tests previously?

Reply:
No, we essentially check `bool(hotStart)` and in that case `None` and `False` behave the same way. I just made this more explicit and to match the docstrings.

Reply:
I think it was just inconsistent with the docstring and the expected input type, but fundamentally did not change anything - see [these lines](https://github.com/mdolab/pyoptsparse/blob/0914abf94f47e26bfbdd36cc83436d7fda5f2c04/tests/testing_utils.py#L317-L318)

Reply:
Makes sense, thanks